doc/tutorial/argument_parser/index.md

```diff
@@ -457,6 +457,15 @@ Add a seqan3::regex_validator to the first positional option that expects the `f
 \snippet doc/tutorial/argument_parser/solution6.cpp file_validator
 \endsolution
 
+# Full solution
+
+Here is the complete code including all the little assignments of this tutorial
+that can serve as a copy'n'paste source for your application.
+
+\htmlonly <div class=\"assignment\"> <details><summary>Click me to show the code</summary> \endhtmlonly
```

Comment:
Why not using `solution` block?

Reply:
Because it is not an actual solution to a specific assignment?

Reply:
But ist the solution block not usable as a standalone block. I mean it is also a solution. Just the full solution. And i thought it was collapsable as well?

Reply:
Yes it is collapsable but it displays
```
> Solution
````
Instead of customizable: "click here to see full code"

Reply:
yeah, but you an just write above `The following soltion shows the complete block`. I would rather not put such a html sutff into md file. I also think that it would be possible to improve the doxygen macro to allow custom messages. Anyway, just using the solution block and adding a sentence like above should suffice for the moment.

Reply:
done